modules/lead/product/toolchain.tf

```diff
@@ -24,6 +24,8 @@ data "template_file" "jenkins_values" {
     appDomain              = "apps.${var.cluster_domain}"
     builder_images_version = var.builder_images_version
     allow_anonymous_read   = var.enable_keycloak ? "false" : "true"
+    jenkins-repository-dockercfg = "jenkins-repository-dockercfg"
```

Comment:
this should be referenced from `kubernetes_secret.jenkins_repository_dockercfg`, not hardcoded